src/main/java/Duke.java

```diff
@@ -2,15 +2,32 @@
 
 public class Duke {
 
+    public static final boolean IS_STARTING = true;
+    public static final boolean IS_ENDING = false;
+
     public static final String DEADLINE_BY_PREFIX = "/by";
     public static final String EVENT_AT_PREFIX = "/at";
     public static final String HELP_MESSAGE = "Valid Commands: " + System.lineSeparator()
             + "todo (description of task)" + System.lineSeparator()
             + "event (description of event) /at (time of event)" + System.lineSeparator()
-            + "deadline (description of task) /by (deadline of task)" +System.lineSeparator()
+            + "deadline (description of task) /by (deadline of task)" + System.lineSeparator()
             + "list" + System.lineSeparator()
             + "bye";
 
+    public static final String LOGO = " ____        _        \n"
```

Comment:
Suggestion:
These text art can be placed into a separate class to reduce the cluttering of code.